/**: # Global Params
  ros__parameters:
      traj_name: "buggycourse_safe.json"

NAND:
  NAND_sim_single:
    ros__parameters:
      velocity: 10
      pose: "Hill1_NAND"

  NAND_controller:
    ros__parameters:
      dist: 0.0
      # traj_name: "buggycourse_safe.json"
      controller: "stanley"

SC:
  SC_sim_single:
    ros__parameters:
      velocity: 12
      pose: "Hill1_SC"

SC:
  SC_controller:
    ros__parameters:
      dist: 0.0
      # traj_name: "buggycourse_safe.json"
      controller: "stanley"

SC:
  SC_path_planner:
    ros__parameters:
      traj_name: "buggycourse_safe.json"
      curb_name: "buggycourse_curb.json"
